OpenRA.Mods.Common/FileFormats/Blast.cs

```diff
@@ -271,7 +272,7 @@ public Huffman(byte[] rep, short symbolCount)
 
 			// no codes!
 			if (Count[0] == n)
-				return;
+				throw new InvalidDataException("No codes for this input");
```

Comment:
This is a bug fix. By this line `Count` is non-null but `Symbol` is still null.

In the `Decode` method both arrays are deferenced without any null checks - it seems that `Decode` would've always failed if we'd ever encountered this "no codes" condition because `Symbol` would be null.